### ISSUE TITLE
Search Functionality.

### ISSUE THREAD

**Issue by mShashank2308** [mShashank2308/hacktoberfest#9]
Add a search functionality to the web page so that the visitor can check if a character is already present or not.

**Comment by Utkrisht12** [comment]
Kindly Assign me this issue, and can I use javascript?
Is this issue valid for Hacktoberfest?

**Comment by mShashank2308** [comment]
Yes you can use javascript. For Hacktoberfest, all you have to do is to create a PR in this repository and it will be counted. Good Luck!
